src/ngAria/aria.js

```diff
@@ -377,6 +377,27 @@ ngAriaModule.directive('ngShow', ['$aria', function($aria) {
             });
           }
         }
+
+        if ($aria.config('bindKeypress') && isNodeOneOf(elem, ['A', 'BUTTON'])) {
```

Comment:
Actually, we don't want to do anything with `button`s, just `a`s. The browser handles pressing both ENTER and SPACE on buttons.